repos/system_upgrade/el7toel8/actors/repositoriesmapping/files/tests/sample01.csv

```diff
@@ -0,0 +1,5 @@
+header 1,header 2,header 3,header 4,header 5,header 6
```

Comment:
If you don't mind, could you add some sample data more descriptive? Like, real title for columns. Content should be mocked (i.e. "SampleRepo01" or something), but with more descriptive data this can be used as some sort of documentation to someone looking to the actor and trying to understand what is expected.